packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -333,6 +344,16 @@ function build(
         // eslint-disable-next-line no-use-before-define
         return queueMessage(vpid, prop, args, returnedP);
       },
+      get(p, prop) {
+        // Support: p~.[prop]
+        lsdebug(`makePipelinablePromise handler.get (${vpid})`);
+        if (!handlerActive) {
+          console.error(`mIPromise handler called after resolution`);
+          assert.fail(X`mIPromise handler called after resolution`);
```

Comment:
Do *not* take this suggesting in this PR. Rather, it is better for this PR to remain consistent with the rest of the code of each module. But it would be good to retire the above convention everywhere, to replace with:

```suggestion
          const err = assert.error(
            X`mIPromise handler called after resolution`,
          );
          console.error(err);
          throw err;
```